library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
> I tried. I had problems defining polymorphic versions.

Oh, I see. But note that I have already constrained `parser` to `Type 1`, since I didn't see any applications for higher universes. At least for parameter parsing, that should be a fair assumption I think.